nvtabular/ops/hash_bucket.py

```diff
@@ -122,6 +123,8 @@ def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None
             new_col = f"{col}_{self._id}"
             if is_list_dtype(gdf[col].dtype):
                 encoded = _encode_list_column(gdf[col], gdf[col].list.leaves.hash_values() % nb)
+            elif gdf[col].dtype == np.int32 or gdf[col].dtype == np.int64:
```

Comment:
This strict dtype checking will miss a bunch of other variants (like uint8 etc). Can we switch this up? I think there are multiple ways of doing this - one way that would work is to check the 'kind' character flag to see if it's a signed/unsigned int:

```suggestion
            elif gdf[col].dtype.kind in ["u", "i"]:
```